prow/bump.sh

```diff
@@ -15,10 +15,23 @@
 
 set -o errexit
 
-if [ "$#" -lt 1 ]; then
-  echo "usage: $0 <program> [program ...]"
-  exit 1
-fi
+# See https://misc.flogisoft.com/bash/tip_colors_and_formatting
+
+color-image() {  # Bold blue
+  echo -e "\e[1;35m${@}\e[0m"
```

Comment:
I think `tput` is more portable if you really care...

Reply:
I really *don't* care :grin: so you're saying I should use this? https://linux.101hacks.com/ps1-examples/prompt-color-using-tput/

Reply:
I did this to handle the case where the script is being pointed at something other an a TTY as well:

https://github.com/openshift/origin/blob/master/hack/lib/util/text.sh

Reply:
I don't really care, just FYI if you do :)

Reply:
we can easily add _more bash_ in a follow up 🤷‍♂️